server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -258,10 +258,15 @@ func (r *statsRecorder) flushInvocationStatsToOLAPDB(ctx context.Context, ij *in
 	}
 	inv, err := r.lookupInvocation(ctx, ij)
 	if err != nil {
-		return status.InternalErrorf("failed to flush invocation stats to clickhouse: %s", err)
+		return status.InternalErrorf("failed to flush invocation stats to clickhouse for invocation id %q: %s", ij.id, err)
 	}
 
-	return r.env.GetOLAPDBHandle().FlushInvocationStats(ctx, inv)
+	err = r.env.GetOLAPDBHandle().FlushInvocationStats(ctx, inv)
+	// Temporary logging for debugging clickhouse missing data.
+	if err != nil {
```

Comment:
do you want `err == nil` here? or change the log to an error message like "failed to write ..." ?

Reply:
intended to have err == nil. The callers already logged the failures